applications/blockops_parachain_node_deployment.md

```diff
@@ -0,0 +1,288 @@
+
+## Team :busts_in_silhouette:
+
+### Team members
+
+- Adedayo Akinpelu - Team Lead: With a compelling 7-year journey in the web3 arena, Dayo has brought his expertise to roles such as Infrastructure Engineer at Flow Blockchain and Quidax Technologies, and Infrastructure Lead at Composable Finance.
+
+- Marcelo Mendoca - Infrastructure Lead: Marcelo, a seasoned Site Reliability Engineer, has charted a dynamic journey within the web3 space, demonstrating his leadership acumen across Software Engineering, DevOps, and Site Reliability.
+
+- Franklin Okpako - Devops & SRE: With over 5 years of experience under his belt, Frank has shown proficiency in the role of a DevOps Engineer. His extensive qualifications include Cloud Native Engineer, CKA, CKS, and AWS Certified DevOps Engineer-Professional.
+
+- Calvin Puram - Devops & SRE: Skilled DevOps engineer with a track record of building scalable applications, optimizing systems, and implementing robust monitoring solutions.
+
+- Oluwatimilehin Adesoji - Software Engineering Lead: Timilehin is a seasoned software engineer with 10+ years of experience building highly available and scalable systems. He is motivated to drive projects/businesses in every way possible through tech.
+
+- John Etokhana - Product Design Lead: As a digital designer across visual identity development, interaction style and cross-platform preparation, John strives to continue his string of successes in the building, testing and refinement of user-centered digital interfaces and experiences.
+
+
+### Contact
+
+- **Contact Name:** Adedayo Akinpelu
+- **Contact Email:** adedayo.akinpelu@blockops.network
+- **Website:** https://blockops.network/
+
+### Legal Structure
+
+- **Registered Address:** 8 The Green, Ste A, Dover DE
+- **Registered Legal Entity:** Blockops Network Inc
+
+### Team's experience
+
+- **Composable Finance**: Setup Networks for Composable Finance for all their chains (Dali, Picasso and Composable)
+- **Polkadot & Kusama**: We have vast experience running RPC and Validator nodes on all Polkadot chains.
+- **Flow Blockchain:** We have experience running Access Nodes and Consensus Nodes on Flow Blockchain
+- We also run Self Managed Nodes (Bitcoin, Ethereum, USDC, USDT etc) for major crypto exchanges in Africa (e.g [Quidax](https://www.quidax.com))
+
+Our team closely monitors the network, stays up to date with upgrades and governance decisions, and adjusts our nodes' configurations accordingly.
+
+Through our experience with Polkadot, we have developed robust strategies to maximize our validator node's performance, optimize staking rewards, and ensure continuous participation in the network's consensus mechanism.
+
+### Team Code Repos
```

Comment:
Since these repos are largely empty, does (anyone on) your team have experience with writing and maintaining open source software that you could share?

Reply:
> Since these repos are largely empty, does (anyone on) your team have experience with writing and maintaining open source software that you could share?

Yes, 3 of our Infrastructure members have this experience but mostly on Contributing and not Maintaining